.github/workflows/unit_tests.yml

```diff
@@ -1,97 +1,131 @@
-    - name: Setup node
-      uses: actions/setup-node@v2-beta
-      with:
-        node-version: ${{ env.NODE_VERSION }}
-    # Basing the cache on the hash of the lockfile means that the cache should
-    # only ever be invalidated when we update package-lock.json, which happens
-    # as part of a release. For more info, see:
-    # https://docs.github.com/en/free-pro-team@latest/actions/guides/caching-dependencies-to-speed-up-workflows
-    - name: Cache node modules
-      uses: actions/cache@v2
-      with:
-        path: ~/.npm
-        key: npm-${{ hashFiles('package-lock.json') }}
-        restore-keys: |
-          npm-${{ hashFiles('package-lock.json') }}
-          npm-
-    - name: Install JS dependencies
-      run: npm ci
-    # Python version to use is stored in the .python-version file, which is the
-    # convention for pyenv: https://github.com/pyenv/pyenv
-    - name: Get Python version
-      run: echo "PYTHON_VERSION=`cat .python-version`" >> $GITHUB_ENV
-    - name: Setup Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ env.PYTHON_VERSION }}
-    # We base the python cache on the hash of all requirements files, so that
-    # if any change, the cache is invalidated.
-    - name: Cache pip
-      uses: actions/cache@v2
-      with:
-        path: ~/.cache/pip
-        key: pip-${{ hashFiles('requirements/*.txt') }}
-        restore-keys: |
-          pip-${{ hashFiles('requirements/*.txt') }}
-          pip-
-    - name: Install dependencies
-      run: pip install -r requirements/test.txt
-    - name: Setup local_settings.py
-      run: |
-        cp ci/testsettings.py cdhweb/local_settings.py
-        python -c "import uuid; print('SECRET_KEY = \'%s\'' % uuid.uuid4())" >> cdhweb/local_settings.py
-    - name: Build static assets
-      run: python manage.py compress
-    - name: Run pytest
-      run: py.test --cov=./ --cov-report=xml
-    - name: Upload test coverage to Codecov
-      uses: codecov/codecov-action@v1
-    # Send a message to slack to report the build status. The webhook is stored
-    # at the organization level and available to all repositories. Only run on
-    # scheduled builds & pushes, since PRs automatically report to Slack.
-    - name: Report status to Slack
-      uses: rtCamp/action-slack-notify@master
-      if: ${{ github.event_name == 'schedule' || github.event_name == 'push' }}
-      env:
-        SLACK_WEBHOOK: ${{ secrets.ACTIONS_SLACK_WEBHOOK }}
-        SLACK_CHANNEL: cdhweb
-        SLACK_TITLE: "Run #${{ github.run_number }}> for workflow `${{ github.workflow }}`: ${{ job.status }}"
-        SLACK_MESSAGE: "<https://github.com/${{ github.repository }}/|${{ github.repository }}@${{ github.ref }}>"
-        SLACK_FOOTER: "<https://github.com/${{ github.repository }}/commit/${{ github.sha }}|View commit>"
-        MSG_MINIMAL: true   # use compact slack message format
+      # Set the value of DJANGO_DB_BACKEND which is used in ci/testsettings.py to
```

Comment:
Why does everything look different here? Did the indentation change? It looks like it is substantially the same; did anything change I should review?

Reply:
several things did change; sorry git isn't highlighting them very well! the main thing to notice is lines 95-101; this tripped me up so I wrote a comment explaining why it's necessary (for mysql only). other than that, I tinkered with the format of the slack message a little, but there aren't substantial changes below this line.

Reply:
ok, I see it now. Thanks for pointing it out.